apps/member-profile/app/shared/components/company-review.tsx

```diff
@@ -47,10 +48,13 @@ type CompanyReviewProps = {
   reviewedAt: string;
   text: string;
   title: string;
+  upvotes?: string | null;
+  upvoted?: boolean | null;
```

Comment:
Is there a particular reason that `upvotes` and `upvoted` are being passed as optionals? It seems to me that we'll always want the like button to be part of this component so it doesn't seem like we need to make it optional

Reply:
I passed them as optionals because if we don't we get the following error when running type-check:

`@oyster/member-profile:type-check: app/routes/_profile.recap.$date.reviews.tsx(79,14): error TS2739: Type '{ key: string; company: { id: string; image: string; name: string; }; date: string; employmentType: EmploymentType; locationCity: string | null; locationState: string | null; ... 9 more ...; title: string; }' is missing the following properties from type 'CompanyReviewProps': upvotesCount, hasUpvoted
@oyster/member-profile:type-check: error Command failed with exit code 2.`

Reply:
I see..so we just want to think about if we would want to have the likes being displayed on the recap page as well. If yes, then we'll have to refactor this (i.e. change that recap file to get the data on likes as well).

Reply:
Hi @tomas-salgado I'm not sure how to see the recap page. I looked around in the code and I don't see it on the website.

Reply:
The recap page is here: https://app.colorstack.io/recap/2024-08-17/leaderboard
It's a little hidden tbh, I don't think there's a way to access it without the direct URL right now.

Reply:
@tomas-salgado I removed the optionals and added the "this was helpful" button to the recap reviews too but made them read-only since it's a recap. Let me know what you think. Thank you!